tests/kuttl/tests/basic-deployment/02-cleanup.yaml

```diff
@@ -4,3 +4,8 @@ delete:
 - apiVersion: core.openstack.org/v1beta1
   kind: OpenStackControlPlane
   name: openstack
+commands:
+- script: |
+    oc delete --ignore-not-found=true -n $NAMESPACE pvc \
+    ansible-ee-logs \
```

Comment:
not sure if I need to delete ansible-ee-logs; it was left over one time locally

Reply:
I think that the `ansible-ee-logs` is created by `make crc_storage` so we don't need to delete it here, but I'm not completely sure about that

Reply:
removed it